domains/bernardoschmitz.json

```
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "B-Schmitz",
+    "email": "bernardo_schmitz@live.com"
+  },
+  "record": {
```

```suggestion
  "records": {
```
it should be records, with "s"

---

sorry, fixed